### ISSUE TITLE
GANTT chart not rendering first data item

### ISSUE THREAD

**Issue by notacouch** [amcharts/amcharts4#889]
In a fork of our [GANTT chart demo](https://www.amcharts.com/demos/gantt-chart-dates/) with 2 categories, each has the same starting time `08:04`:

```javascript
[
  {
    "name": "Category A",
    "fromDate": "2019-01-23 08:04",
    "toDate": "2019-01-23 09:04",
    "color": "green"
  },
  {
    "name": "Category B",
    "fromDate": "2019-01-23 08:04",
    "toDate": "2019-01-23 09:06",
    "color": "red"
  },
  // ...
]
```

Yet Category A does not print its first column, though Category B does:

![image](https://user-images.githubusercontent.com/2334143/51715225-d9a67180-2005-11e9-98bc-d8d807a83aa3.png)

https://codepen.io/team/amcharts/pen/c6164b32ca12a1fc0db0e70a0e24e666

If we swap those 2 data items so Category B comes first, somehow both print/render:

![image](https://user-images.githubusercontent.com/2334143/51715281-09557980-2006-11e9-9280-4635e5477477.png)

https://codepen.io/team/amcharts/pen/1ddac05119e1df930f3b43849593275c

**Comment by sQty** [comment]
in my case  i dont have 2 points with equal start date 

i have  just 2 categories

chart.data = [
  {
    "name": "Cat A",
    "fromDate": "2019-01-23 08:05",
    "toDate": "2019-01-23 09:04",
    "color": "green"
  },
  {
    "name": "Cat B",
    "fromDate": "2019-01-23 09:04",
    "toDate": "2019-01-23 09:06",
    "color": "red"
  },
  {
    "name": "Cat A",
    "fromDate": "2019-01-23 09:06",
    "toDate": "2019-01-23 10:42",
    "color": "green"
  },
  {
    "name": "Cat B",
    "fromDate": "2019-01-23 10:42",
    "toDate": "2019-01-23 11:02",
    "color": "red"
  },
  {
    "name": "Cat A",
    "fromDate": "2019-01-23 11:02",
    "toDate": "2019-01-23 13:14",
    "color": "green"
  },
  {
    "name": "Cat B",
    "fromDate": "2019-01-23 13:14",
    "toDate": "2019-01-23 13:19",
    "color": "red"
  },
  {
    "name": "Cat A",
    "fromDate": "2019-01-23 13:19",
    "toDate": "2019-01-23 16:08",
    "color": "green"
  }
];

workaround
add counter to each obj name  

chart.data = [
  {
    "name": "1. Cat A",
    "fromDate": "2019-01-23 08:05",
    "toDate": "2019-01-23 09:04",
    "color": "green"
  },
  {
    "name": "2. Cat B",
    "fromDate": "2019-01-23 09:04",
    "toDate": "2019-01-23 09:06",
    "color": "red"
  },
  {
    "name": "3. Cat A",


.........

**Comment by martynasma** [comment]
Fixed in [4.1.2](https://github.com/amcharts/amcharts4/releases/tag/4.1.2).

## [4.1.2] - 2019-02-05

### Added
- New `Sprite` property `strokeDashoffset` added. Can be used in conjunction with `strokeDasharray`.
- `Label` has two new properties: `path` and `locationOnPath`. If set, will layout the SVG text along a curvature of the SVG path. HTML and multi-line text is not supported.
- `AxisLabelCircular` new property `bent` added. If set to `true`, the label will be bent to follow the curvature of the circle. Distance from the circle can be adjusted using `radius` and `label.paddingBottom`. For `PieSeries` the alignLabels need to to be set to `false` for this feature to work.

### Fixed
- JSON: Having property values with both percent and minus signs in them, was resulting in error.
- "Z" code in `DateFormatter` was not taking UTC setting into account.
- Exporting SVG with Unicode characters in Edge/IE was resulting in invalid SVG.
- Fixed JavaScript error which used happen when disposing chart/changing data in some cases.
- Gantt chart sometimes was not displaying first/last data item.
- Newly added `MapLine` elements were not respecting `nonScalingStroke = true` until zoom level changed.
- Series' tooltip was not always visible with `DateAxis`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.

**Comment by Zahar661** [comment]
if someone is looking for a similar problem from version 5, sort the categories and data the same way.
